scripts/core/utils.ts

```diff
@@ -32,3 +32,54 @@ export const getSuperdeskType = (event, supportExternalFiles = true) =>
     event.originalEvent.dataTransfer.types.find((name) =>
         name.includes('application/superdesk') || supportExternalFiles && name === 'Files',
     );
+
+    /**
+ * @ngdoc method
+ * @name gettext
+ * @param {String} text - the text that will be translated, it supports parameters (see the example)
+ * @param {Object} params - dictionary of parameters used in text and their value (see the example)
+ * @description Used angular gettext service for displaying localised text on Browser
+ *
+ * gettext('This item was locked by {{username}}.', {username: 'John'});
+ * result -> 'This item was locked by John'
+ */
+export const gettext = (text, params = null) => {
+    if (!text) {
+        return '';
+    }
+
+    const injector = angular.element(document.body).injector();
+
+    if (injector) { // in tests this will be empty
+        return injector.get('gettextCatalog').getString(text, params || {});
+    }
+
+    return text;
+};
+
+/**
+ * @ngdoc method
+ * @name gettextPlural
+ * @param {Number} count
+ * @param {String} text
+ * @param {String} pluralText
+ * @param {Object} params
+ * @description Used angular gettext service for displaying plural localised text on Browser
+ */
+const gettextPlural = (count, text, pluralText, params) => {
```

Comment:
`params` argument shouldn't be required

Reply:
OK, I can add default value:
const gettextPlural = (count, text, pluralText, params={}) => {